src/Illuminate/Auth/Access/Middleware/Authorize.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Illuminate\Auth\Access\Middleware;
+
+use Closure;
+use Illuminate\Contracts\Auth\Access\Gate;
+
+class Authorize
+{
+    /**
+     * The gate instance.
+     *
+     * @var \Illuminate\Contracts\Auth\Access\Gate
+     */
+    protected $gate;
+
+    /**
+     * Create a new middleware instance.
+     *
+     * @param  \Illuminate\Contracts\Auth\Access\Gate  $gate
+     * @return void
+     */
+    public function __construct(Gate $gate)
+    {
+        $this->gate = $gate;
+    }
+
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @param  string  $ability
+     * @param  string|null  $model
+     * @return mixed
+     *
+     * @throws \Illuminate\Auth\Access\AuthorizationException
+     */
+    public function handle($request, Closure $next, $ability, $model = null)
+    {
+        $this->gate->authorize($ability, $this->getGateArguments($model));
```

Comment:
@mark86092 thanks for catching that!

Just goes to show that you always need tests. So I now wrote a test for the middleware too.